tests/mocha/service.js

```diff
@@ -1,7 +1,54 @@
 import assert from 'assert'
 
-describe('my feature', () => {
-    it('should do stuff', () => {
+const sleep = (ms) => new Promise(resolve => setTimeout(resolve, ms))
+
+before(async () => {
+    await sleep(10)
+})
+
+describe('my feature 1', () => {
```

Comment:
I wonder if we should better structure these smoke tests. This test here was suppose to check of commands are being run synchronously. I think it makes sense to create an extra test file that tests the hooks.